flocker/dockerplugin/__init__.py

```diff
@@ -5,3 +5,13 @@
 
 This may eventually be a standalone package.
 """
+
+from twisted.internet.address import UNIXAddress
+
+# Many places in both twisted.web and Klein are unhappy with listening on
+# Unix socket, e.g.  https://twistedmatrix.com/trac/ticket/5406.  The
+# Docker plugin needs to listen using a Unix socket, so "fix" that by
+# pretending we have a port number and host. Yes, I feel guilty.
+UNIXAddress.port = 0
+UNIXAddress.host = b"127.0.0.1"
+del UNIXAddress
```

Comment:
I'm actually a little concerned about what effect this might have on the validity of the rest of the Flocker test suite.  This code will be loaded into a `trial flocker` process but not into an *actual* Flocker process.  Are we going to mask other bugs in Flocker with this?

Fortunately I don't think we run any other Flocker services over a UNIX socket ... at the moment ... But if we did, maybe these lines would make it appear as though it works when it really won't.